File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/scaladoc/lexer/ScalaDocLexerTest.java
package org.jetbrains.plugins.scala.lang.scaladoc.lexer;

import com.intellij.lang.Language;
import junit.framework.Test;
import junit.framework.TestCase;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.plugins.scala.lang.lexer.LexerTestBase;
import org.jetbrains.plugins.scalaDoc.ScalaDocLanguage;

public class ScalaDocLexerTest extends TestCase {
    @NotNull
    public static Test suite() {
        return new LexerTestBase("/lexer/scalaDocData") {
            @NotNull
            @Override
            protected Language getLanguage() {
                return ScalaDocLanguage.INSTANCE;
            }

            @Override
            protected void printTokenRange(int tokenStart, int tokenEnd, @NotNull StringBuilder builder) {
            }
        };
    }
}

File: scala/scala-impl/test/org/jetbrains/plugins/scala/lang/scaladoc/lexer/ScalaDocLexerTest.scala
package org.jetbrains.plugins.scala.lang.scaladoc.lexer

import com.intellij.lang.Language
import org.jetbrains.plugins.scala.lang.lexer.LexerTestBase
import org.jetbrains.plugins.scalaDoc.ScalaDocLanguage

import java.nio.file.Path

class ScalaDocLexerTest extends LexerTestBase {
  override protected def relativeTestDataPath: Path = Path.of("lexer", "scalaDocData")

  override protected def language: Language = ScalaDocLanguage.INSTANCE

  override protected def printTokenRange(tokenStart: Int, tokenEnd: Int, builder: StringBuilder): Unit = {}
}
